src/main/graphql/GetFileCheckProgress.graphql

```diff
@@ -0,0 +1,10 @@
+query getFileCheckProgress($consignmentId: UUID!) {
+    getConsignment(consignmentid: $consignmentId) {
+        totalFiles
+        fileCheckProgress {
```

Comment:
Do these names need to be updated to match the latest version of [ConsignmentFields](https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala)?

Reply:
They do! Sorry totally missed that :sweat_smile: